odmObject/odmobject-ipso-temperature.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
can we remove this one from this pull request, and add changes to the existing OCF temperature?

Reply:
I propose to keep both for now (it is the playground after all), in order to look at how to do both odm adoption process and how the handle mappings file. E.g. for temperature the differences how range is indicated.